exporter/datadogexporter/internal/model/attributes/gcp/gcp.go

```diff
@@ -43,9 +43,10 @@ func HostInfoFromAttributes(attrs pcommon.Map) (hostInfo *HostInfo) {
 	hostInfo = &HostInfo{}
 
 	if hostID, ok := attrs.Get(conventions.AttributeHostID); ok {
-		// Add host id as a host alias to preserve backwards compatibility
-		// The Datadog Agent does not do this
-		hostInfo.HostAliases = append(hostInfo.HostAliases, hostID.StringVal())
+		if cloudAccount, ok := attrs.Get(conventions.AttributeCloudAccountID); ok {
+			alias := fmt.Sprintf("%s.%s", hostID.StringVal(), cloudAccount.StringVal())
+			hostInfo.HostAliases = append(hostInfo.HostAliases, alias)
+		}
```

Comment:
Note for future git history spelunkers: this is not a breaking change since we were not sending host aliases at all to the backend.